src/store/metadata/stub/store.py

```diff
@@ -57,6 +57,7 @@ def setup(self):
         using the contents of ./content
         """
         content_dir = Path(Path(__file__).parent / "content")
+        context_dir = Path(Path(__file__).parent.parent)
```

Comment:
so the code is good but I wouldn't put it in the store class.

so it works, but there's no Oxigraph version of context (a populated graph doesnt care about a jsonld context file) so we'd have to move the context logic somewhere ekse in order to delete the stub store (which we'll be doing relatively soon).

I would probably just add a context.py at the root level for now.

Reply:
OK, this is a bit of a weird one - I tried moving `get_context()` to a `context.py` file and calling the function directly in `get_context()` in `main.py`, but I was getting a failure for `assert_called_once()` in the tests (i.e. nothing was being called). So I created a new class called `ContextStore` (currently in `base.py`), so that I could include it as a dependency, which fixed the problem. Let me know if you think there's a better way to do it.